airbyte-config/models/src/main/java/io/airbyte/config/helpers/GcsLogs.java

```diff
@@ -27,19 +26,28 @@
   private static final Logger LOGGER = LoggerFactory.getLogger(GcsLogs.class);
 
   private static Storage GCS;
```

Comment:
@davinchia is there any reason we are using a static here and in S3Logs? or if just pass in the client itself is that fine?

Reply:
mostly because we want to reuse the Cloud Storage clients if possible since these can be heavyweight.

I'm fine with passing in if you think it improves readability. This is mostly from experience working on Dist but we aren't hitting those lives of storage use yet so this is partially premature.

Reply:
Although I definitely don't think we should be creating these everything we retrieve logs so definitely favor reusing.

Reply:
got it! makes sense. i'll keep it.